pmlc/ast/builder.cc

```diff
@@ -738,6 +739,38 @@ struct ProgramBuilder {
         .result();
   }
 
+  Value makeGatherOp(ExprNodeIntrinsic *node, ArrayRef<Value> operands) {
+    TensorShape shape = evaluator.getShape(node);
+    RankedTensorType resultType = builder.getRankedTensorType(shape);
+    auto op = builder.create<tile::GatherOp>(
+        loc, resultType, ValueRange{operands[0], operands[1]},
```

Comment:
Use `operands.take_front(2)` instead of `ValueRange{operands[0], operands[1]}`

Reply:
Fixed